README.md

```diff
@@ -69,7 +92,7 @@ A TypeScript project will have separate `src` and `dist` directories, for source
 |test/|this directory contains your tests|
 |tsconfig.json|the configuration settings for TypeScript compilation|
 
-### TypeScript Configuration
+### <a name="typescriptConfiguration"></a>TypeScript Configuration
 
 [TODO]
 In your `tsconfig.json` file, you can specify options for TypeScript to use when compiling your project. 
```

Comment:
Once merged @emckean will work on README.  I've communicated this point with @emckean .